Shokofin/Resolvers/VirtualFileSystemService.cs

```diff
@@ -1179,6 +1265,31 @@ private LinkGenerationResult CleanupStructure(string vfsPath, string directoryTo
                 result.RemovedPaths.Add(location);
                 result.RemovedSubtitles++;
             }
+            else if (AudioFileExtensions.Contains(extName)) {
+                if (TryMoveAudioFile(allKnownPaths, location, preview, out var skip)) {
```

Comment:
Don't create an exact duplicate function of `TryMoveSubtitleFile`, and instead rename and adapt `TryMoveSubtitleFile` to indicate it works with both subtitle files and audio files. And by adapt I mean tweak the variable names.